ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -39,11 +42,13 @@ export class PeerConnectionManager {
     logger: Logger = createRootLogger(),
     options: {
       maxPeers: number
+      keepOpenPeerSlot?: boolean
```

Comment:
Alternative: `keepOpenPeerSlot: boolean = DEFAULT_KEEP_OPEN_PEER_SLOT`

Reply:
Type literals can't have an initializer: 
![image](https://github.com/iron-fish/ironfish/assets/97762857/96db30d4-a58f-4a48-a4a5-7d19ba1979b4)


You can kinda emulate this behavior but it's a lot more verbose than the existing pattern. We do it somewhere else but I can't remember where off the top of my head